demo/lightning/image-segmentation/train.py

```diff
@@ -51,8 +51,15 @@ def init_processes():
     if flags.benchmark:
         profiler = "simple"
 
-    model = Unet3DLightning(flags)
+    listing_start = time.time()
     train_data_loader = Unet3DDataModule(flags)
+    listing_end = time.time()
+    if flags.listing_only:
+        print(
```

Comment:
nit: Should we print the list time regardless?

Reply:
Added "Passing `--benchmark` along with `--listing_only` would have the same effect as passing just the latter."

Reply:
Updated so that list time is printed regardless of the mode. The user is notified that training is being skipped when `--listing_only` is passed: http://shortn/_wTll5QX02T